### DISCUSSION TITLE
Clarify the scope of use and access security of the submitted model

### DISCUSSION THREAD

**Discussion by Cryolite** [smly/mjai.app#7] [General]
Thank you for your excellent service.

For the purpose of this service, I assume that submitted models will not be used for anything other than playing against other models, and that the submitted models will not be made public to any third party. I would like to see this clearly stated in the documentation.

I believe that many mahjong AI authors want their AI to play against other AIs, but do not want their models to be available to the public because they do not want their models to be used for cheating or other undesirable purposes. I think the above request is important to allay concerns that the models submitted to this service may have such potential.

**Comment by smly** [comment]
You are very right. The policy clarification is the most important issue that mjai.app lacks at this stage. I also recognize that having a strong mahjong AI publicly available is good for learning and competition, but it could be used for inappropriate purposes.

At the very least, I will make it clear that SUBMITTED MODELS will not be used or published for any other purpose.
I will be taking a look at the policies of several services and will make this clear as soon as possible.

Thanks for the great feedback! If anyone else has an opinion on this subject, please feel free to comment :-)

**Reply by smly** [reply]
At first, I added message to upload screen.
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/19337/193385771-36d54884-b1c5-4dbd-82eb-ce6f94633247.png">
